### PR TITLE
zh-trans: add kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md

### PR DESCRIPTION
**Pull request by AdamDang**
zh-trans: add kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md

### CODE REVIEW

**Comment by xichengliudui** [comment]
@AdamDang  @xichengliudui and @zgfh also translate files in the current directory. I suggest you check our PR to see if it has been translated, so as not to waste time.
[zgfh](https://github.com/kubernetes/website/pulls/zgfh)
[xichengliudui ](https://github.com/kubernetes/website/pulls/xichengliudui)
thanks

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
+<!--
+renew all available certificates
+
+### Synopsis
+
+
+Renews all known certificates necessary to run the control plan. Renewals are run unconditionally, regardless of expiration date. Renewals can also be run individually for more control.
+-->
+
+更新所有可用的证书
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md]: Done!

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+### 摘要
+
+
+更新运行控制计划所需的所有已知证书。无论到期日期如何，更新都是无条件运行的。更新也可以单独运行以获得更多的控制。
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md]: Done！

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+<table style="width: 100%; table-layout: fixed;">
+  <colgroup>
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">--cert-dir string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes/pki"</td>
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md]: Done！

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">kubeadm 配置文件的路径（警告: 配置文件的使用是实验性的）</td>
+    </tr>
+
+    <tr>
+      <td colspan="2">-h, --help</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">所有帮助信息</td>
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md]: Done！

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+    </tr>
+
+    <tr>
+      <td colspan="2">--kubeconfig string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: "/etc/kubernetes/admin.conf"</td>
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md]: Done！

File: content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+    <col span="1" style="width: 10px;" />
+    <col span="1" />
+  </colgroup>
+  <tbody>
+
+    <tr>
+      <td colspan="2">--rootfs string</td>
+    </tr>
+    <tr>
+      <td></td><td style="line-height: 130%; word-wrap: break-word;">[实验]通往'真正的'主机 root 文件系统的路径。</td>
```
[the remark this answers is not in the document]
Comment (AdamDang) [reply content/zh/docs/reference/setup-tools/kubeadm/generated/kubeadm_alpha_phase_certs_renew_all.md]: Done！

**Comment by AdamDang** [comment]
@xichengliudui @zgfh
OK, Please update the issue status before translating. I  just checking the issue status (pending) and the target directory (translated file not exist).

**Comment by xichengliudui** [comment]
I'm sorry if it bothers you.

**Comment by tengqm** [comment]
@AdamDang Please check if the translation of "renew" is consistent across the page.

**Comment by Hartmarken348** [comment]
Alpha certificates

**Pull request merged**